Source/Classes/Provider/ClassificationNode.cs

```diff
@@ -14,6 +14,7 @@ public class ClassificationNode : Leaf
       public bool HasChildren { get; set; }
       public string StructureType { get; set; }
       public string ParentUrl => this.Links?.Parent;
+      public ClassificationNodeAttributes Attributes { get; }
```

Comment:
Somehow I am a bit blocked. Could you explain this addition shortly for me?

Reply:
ClassificationNodes have an object property of attributes. This object property has two properties itself - `startDate` and `endDate`. When an iteration has dates assigned to it, those attributes are populated with the dates. The current implementation requires a user to traverse the original object to get the `startDate` and `endDate`. I've exposed those here to alleviate this requirement.